.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+jobs:
```

Comment:
How would this work if we want to add builds for multiple JDKs as I did with Travis? Would we need multiple jobs? If so, is there a way to do it without massive duplication?

Reply:
I'm not sure TBH, this is the first time I use CirceCI